content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -30,32 +30,29 @@ your specific needs. The following built-in configuration profiles are currently
 
 1. **minimal**: the minimal set of components necessary to use Istio's [traffic management](/docs/tasks/traffic-management/) features.
 
-1. **sds**: similar to the **default** profile, but also enables Istio's [SDS (secret discovery service)](/docs/tasks/security/citadel-config/auth-sds).
-    This profile comes with additional authentication features enabled by default (Strict Mutual TLS).
-
 1. **remote**: used for configuring remote clusters of a
     [multicluster mesh](/docs/ops/deployment/deployment-models/#multiple-clusters) with a
     [shared control plane](/docs/setup/install/multicluster/shared-vpn/) configuration.
 
+1. **preview**: this profile enables experimental features that may not meet the same level of stability, performance, and security as the default profiles.
+
+1. **empty**: deploys nothing. This can be useful as a base profile for custom configuration.
```

Comment:
The phrase `deploys nothing.` can be confusing for readers. They might ask: Will the command do *nothing*? Which is not what happens. This profile just means that no configuration profile is applied, correct? The text should let readers know that's the case.

Reply:
It literally does nothing. Its not the same as no profile, which would mean the default one. It was added so you can do things like `--set profile=empty --set something.enabled=true` instead of needing to go and disable all the default stuff.

I didn't want to document it since its confusing but I got recommended to add all profiles here

Reply:
I think instead of `deploys nothing`,  the following would be better.  

`The empty profile has everything disabled.`